resources/views/layouts/master.blade.php

```diff
@@ -6,6 +6,11 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    @if (isset($global_vars->custom_font_embed) || isset($global_vars->custom_font_name))
+      {!! $global_vars->custom_font_embed !!}
+      <style type="text/css"> * { font-family: "{{ $global_vars->custom_font_name }}"; } .heading { font-family: "{{ $global_vars->custom_font_name }}" !IMPORTANT; font-weight:700 !IMPORTANT;}</style>
```

Comment:
It might be safer to apply this `* { ... }` directly to the `body { }` tag so it just overrides that original declaration (and then it _should_ cascade anywhere it's not overridden). The `*` selector will apply that rule directly to every element on the page.